ads/_ping_.js

```diff
@@ -47,6 +47,12 @@ export function _ping_(global, data) {
     } else {
       context.renderStart();
     }
+    window.context.observeIntersection(function(changes) {
```

Comment:
I believe `window.context` works, and we documented that way. But since we use the name global here, should we call them `global.context` instead? 
And it's pretty weird that `context` also works is it `window` is by default?